.papr.sh

```diff
@@ -121,11 +132,11 @@ fi
 # Run integration tests
 if [ $integrationtest -eq 1 ]; then
     make TAGS="${TAGS}" test-binaries
-    SKIP_USERNS=1 make varlink_generate GOPATH=/go
+    make varlink_generate GOPATH=/go
     if [ $runpython -eq 1 ]; then
-        SKIP_USERNS=1 make clientintegration GOPATH=/go
+        make clientintegration GOPATH=/go
```

Comment:
do you not need the `$INTEGRATION_TEST_ENVS` bit here and at line 135?

Reply:
no, the env variable is only for integration tests.

Reply:
Not bare metal - Travis runs in a container right now